src/modules/help.ts

```diff
@@ -0,0 +1,85 @@
+import { command, default as CookiecordClient, Module } from 'cookiecord';
+import { Message, MessageEmbed } from 'discord.js';
+
+export default class HelpModule extends Module {
+	protected client: any;
+	constructor (client: CookiecordClient) {
+		super(client);
+
+		this.client = client;
+	}
+
+	@command({ aliases: ['help', 'commands', 'h'] })
+	async help (message: Message, args: string) {
```

Comment:
Cookiecord will only give you one word with this code, read up on [this](https://cookiecord.js.org/#single-arg-mode) if you want to avoid the automatic argument parsing.

Rename `message` to `msg` for consistency.